.env.example

```
@@ -1,9 +1,59 @@
 ##################################
 # ********* App Config ********* #
 ##################################
+
 # App Configuration
-NEXT_PUBLIC_APP_URL=""
-NEXT_PUBLIC_SHOW_LOGGER=""
-# Strapi Configuration
-NEXT_PUBLIC_STRAPI_API_URL=""
+# The public URL of your application (used for redirects and email links)
+NEXT_PUBLIC_APP_URL="http://localhost:3000"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)